### DISCUSSION TITLE
What is the path forward for OpenBao?

### DISCUSSION THREAD

**Discussion by boldandbusted** [openbao/openbao#96] [General]
Howdy. I'm pretty excited about the progress at OpenTofu. I'm eager to see OpenBao also make its mark. I have several questions that I hope will facilitate discussion.

* Is there any substantial support or interest from companies that integrate Vault into their products supporting OpenBao (financial, donated labor, legal, etc.)?
* Will OpenBao have a foundation to hold trademarks, handle legal matters, contributions, etc.?
* Will OpenBao aim for inclusion in the Linux Foundation or CNCF umbrella organizations?

Yeah, big questions. Apologies if this is addressed well elsewhere, but do please include links to discussions here so forum-members can see it more easily. Thank you! I'm eager to see OpenBao GA!

**Comment by joewxboy** [comment]
@boldandbusted good questions.  I'll try my best to answer them.

1. Yes.  That is the pool we primarily recruited the founding organizations from: those that use or plan to use OpenBao, those that plan to integrate with OpenBao, and those that provide services that are compatible with OpenBao.  The founding organizations are listed at https://wiki.lfedge.org/display/OH/OpenBao+%28Hashicorp+Vault+Fork+effort%29+support

2. and 3. Absolutely, yes.  Our [mission statement](https://wiki.lfedge.org/display/OH/OpenBao+%28Hashicorp+Vault+Fork+effort%29+FAQ) is: 

> OpenBao exists to provide a software solution to manage, store, and distribute sensitive data including secrets, certificates, and keys.  The OpenBao community intends to provide this software under an OSI-approved open-source license, led by a community run under open governance principles.

The TSC (Technical Steering Committee) plans to vote on which specific foundation to join on or after February 8th, after completing due diligence.  Our goal is a unanimous decision, if possible.  See the [recording and slides from our meetings](https://wiki.lfedge.org/display/OH/OpenBao+Meetings). Today's community status call discussed this briefly.

**Reply by boldandbusted** [reply]
Thank you so much for fielding my pesky questions! 🌞

**Reply by joewxboy** [reply]
No worries.  Please post if you have any more.  If nothing else, it helps us to find out where we are not communicating well and address those shortcomings. :)

**Comment by cipherboy** [comment]
As of last week's meeting, the TSC was [officially formed](https://wiki.lfedge.org/display/OH/2024-06-13+OpenBao+TSC+Meeting) of members [discussed in the CONTRIBUTING.md](https://github.com/openbao/openbao/blob/main/CONTRIBUTING.md#technical-steering-committee-tsc-members) document. At the same meeting, [maintainers](https://github.com/openbao/openbao/blob/main/MAINTAINERS.md) were elected. OpenBao remains incubating under LF Edge. 

Hope this helps to answer the questions!
